test/unit/test_query_context_cache.py

```diff
@@ -227,64 +228,64 @@ def test_same_id_with_stale_timestamp(
     assert_cache_with_data(qcc_with_data, expected_data)
 
 
-@pytest.mark.skipif(not installed_pandas, reason="pyarrow not installed")
+@pytest.mark.skipif(not installed_pandas, reason="pandas not installed")
```

Comment:
all `pytest.mark.skipif` can be removed that are checking for `installed_pandas` since we are not using arrow/pyarrow ser-deser.

Reply:
I agree. These changes aimed to bypass the pyarrow installation check. I can remove all related checks.